### PR TITLE
Fix a bug in annotation preparation

### PR DESCRIPTION
**Pull request by jiangliu**
Fix a bug in annotation preparation and also refine annotation names.

### CODE REVIEW

File: pkg/label/label.go [hunk 1]
```diff
@@ -7,21 +7,24 @@
 package label
 
 const (
-	Signature = "containerd.io/snapshot/nydus-signature"
+	// Labels defined by CRI
+	CRIImageRef            = "containerd.io/snapshot/cri.image-ref"
+	CRIImageLayers         = "containerd.io/snapshot/cri.image-layers"
+	CRILayerDigest         = "containerd.io/snapshot/cri.layer-digest"
+	CRIManifestDigestLabel = "containerd.io/snapshot/cri.manifest-digest"
```
Comment (imeoer) [inline pkg/label/label.go:14]: Maybe `CRIManifestDigest`.

File: pkg/label/label.go [hunk 2]
```diff
@@ -7,21 +7,24 @@
...
-	ImagePullSecret   = "containerd.io/snapshot/pullsecret"
-	ImagePullUsername = "containerd.io/snapshot/pullusername"
-
-	TargetSnapshotLabel       = "containerd.io/snapshot.ref"
-	TargetManifestDigestLabel = "containerd.io/snapshot/cri.manifest-digest"
-	StargzLayer               = "containerd.io/snapshot/stargz"
+	// Labels defined by containerd
+	ImagePullSecret     = "containerd.io/snapshot/pullsecret"
+	ImagePullUsername   = "containerd.io/snapshot/pullusername"
+	TargetSnapshotLabel = "containerd.io/snapshot.ref"
```
Comment (imeoer) [inline pkg/label/label.go:19]: Maybe `TargetSnapshotRef`.

File: pkg/label/label.go [hunk 3]
```diff
@@ -7,21 +7,24 @@
...
-
-	ImagePullSecret   = "containerd.io/snapshot/pullsecret"
-	ImagePullUsername = "containerd.io/snapshot/pullusername"
-
-	TargetSnapshotLabel       = "containerd.io/snapshot.ref"
-	TargetManifestDigestLabel = "containerd.io/snapshot/cri.manifest-digest"
-	StargzLayer               = "containerd.io/snapshot/stargz"
+	// Labels defined by containerd
+	ImagePullSecret     = "containerd.io/snapshot/pullsecret"
+	ImagePullUsername   = "containerd.io/snapshot/pullusername"
```
Comment (imeoer) [inline pkg/label/label.go:18]: `ImagePullSecret` and `ImagePullUsername` are defined by nydus. :)

**Comment by imeoer** [comment]
Tested with https://github.com/dragonflyoss/image-service/blob/master/contrib/ctr-remote/commands/rpull.go#L97, it works fine.

File: pkg/label/label.go (hunk 3 above)
Comment (jiangliu) [reply pkg/label/label.go:18]: So add Nydus prefix？

Comment (imeoer) [reply pkg/label/label.go:18]: Looks good to me.

File: pkg/label/label.go (hunk 1 above)
Comment (jiangliu) [reply pkg/label/label.go:14]: Fixed

File: pkg/label/label.go (hunk 2 above)
Comment (jiangliu) [reply pkg/label/label.go:19]: Fixed

**Review by imeoer — APPROVED** [review]

**Pull request merged**